src/finance_quote/app.py

```diff
@@ -5,20 +5,25 @@
 """
 import logging
 from typing import List
-
+from finance_quote.modules.fixerio import Fixerio
 
 class App:
     """ The main entry point to the F::Q library """
     def __init__(self):
         self.logger = logging.getLogger(__name__)
         self.modules = ['alphavantage']
 
-    def download(self, provider: str, symbols):
+    def download(self, provider: str, symbols, currency: str):
```

Comment:
the function does not return anything, is this expected ? is it acting through some side-effect ?

Reply:
Fixed, thanks!